src/views/BrokeringRuns.vue

```diff
@@ -175,8 +176,20 @@ async function setEComStore(event: CustomEvent) {
   emitter.emit("dismissLoader")
 }
 
-function getScheduleFrequency(cronExp: string) {
-  return Object.entries(cronExpressions).find(([description, expression]) => expression === cronExp)?.[0] || "-"
+function getScheduleFrequency(brokeringGroupObj: any) {
+  let cronDescription = "-";
+  let description = Object.entries(cronExpressions).find(([description, expression]) => expression === brokeringGroupObj.cronExpression)?.[0]
```

Comment:
We should not use Object.entries and instead of values or keys as required as description here is unused.